crypto/CMakeLists.txt

```diff
@@ -142,7 +142,14 @@ if (CONFIG_NRF_CC3XX_PLATFORM)
     set (CC3XX_ARCH cc310)
   endif()
 
-  set (NRF_CC3XX_PLATFORM_BASE ${CMAKE_CURRENT_SOURCE_DIR}/nrf_${CC3XX_ARCH}_platform)
+  if (DEFINED ENV{CUSTOM_CC3XX_BUILD_PATH})
```

Comment:
we should not suddenly check for an environment variable that suddenly change the code path.

This setting is not documented anywhere, and if we start using this principle here then other part of the NCS codebase will soon copy this principle to other places, making it much harder to understand how to use / configure NCS.

Please ping directly for discussion of alternative solutions depending on exact needs.